apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx

```diff
@@ -36,17 +42,36 @@ export const ConnectWalletButton: FC<
     );
   } else {
     return (
-      <WalletIdentity
-        onDisconnect={onDisconnect}
-        address={address}
-        dataLayoutId={dataLayoutId}
-        className={className}
-        balance={<WalletBalance />}
-        submenuLabels={{
-          copyAddress: 'Copy Address',
-          disconnect: 'Disconnect',
-        }}
-      />
+      <>
+        <WalletIdentity
+          onDisconnect={onDisconnect}
+          address={address}
+          dataLayoutId={dataLayoutId}
+          className={className}
+          balance={
```

Comment:
This is weird, can you confirm with Pili if we don't want to have balance for the first iteration and if so, rename the prop appropriately? It just seems weird to assign all of the links to `balance`.

Reply:
yep, it was confirmed with phili that we will not show balance anymore, at least at first. I will change prop name to something that could contain any kind of stuff, maybe `content`